saleor/plugins/openid_connect/plugin.py

```diff
@@ -114,6 +116,14 @@ class OpenIDConnectPlugin(BasePlugin):
             ),
             "label": "OAuth logout URL",
         },
+        "audience": {
+            "type": ConfigurationTypeField.STRING,
+            "help_text": (
+                "The Oauth resource identifier. If provided, Saleor will define "
```

Comment:
I think the name is OAuth :) 
```suggestion
                "The OAuth resource identifier. If provided, Saleor will define "
```